scripts/linters/excluded_any_type_files.json

```diff
@@ -87,6 +87,7 @@
     "core/templates/services/schema-undefined-last-element.service.ts",
     "core/templates/services/site-analytics.service.ts",
     "core/templates/services/speech-synthesis-chunker.service.ts",
+    "core/templates/services/state-interaction-stats.service.ts",
```

Comment:
In general files shouldn't be added to this list. Adding @nishantwrp to advise if this is OK to submit.

Reply:
Obsolete since I've given everything a non-any type.

Reply:
Obsolete; everything has been typed so we no longer need to add the file.